staging/src/k8s.io/kubectl/pkg/drain/drain.go

```diff
@@ -445,6 +445,11 @@ func waitForDelete(params waitForDeleteParams) ([]corev1.Pod, error) {
 		pods = pendingPods
 		return len(pods) == 0, nil
 	})
+
+	if err != nil {
+        return pods, wait.ErrorInterrupted(err)
+    }
```

Comment:
Is this a display problem or a formatting problem?

Reply:
it's a formatting issue, fixed now